junifer/markers/falff/falff_base.py

```diff
@@ -146,7 +146,7 @@ def _compute(
             estimator = JuniferALFF()
         # Compute ALFF + fALFF
         alff, falff, alff_path, falff_path = estimator.compute(  # type: ignore
-            data=input_data["data"],
+            input_path=input_data["path"],
```

Comment:
Is there any reason why we move to paths instead of in-memory data?

Reply:
Passing the path is easier to reason about inside the method and cheaper to hash for the LRU cache.